easybuild/easyconfigs/m/Mikado/Mikado-2.3.4-foss-2022b.eb

```
@@ -0,0 +1,85 @@
+easyblock = 'PythonBundle'
+
+name = 'Mikado'
+version = '2.3.4'
+
+homepage = 'https://github.com/EI-CoreBioinformatics/mikado'
+description = """Mikado is a lightweight Python3 pipeline to identify
+ the most useful or “best” set of transcripts from multiple transcript
+ assemblies. Our approach leverages transcript assemblies generated
+ by multiple methods to define expressed loci, assign a representative
+ transcript and return a set of gene models that selects against transcripts
+ that are chimeric, fragmented or with short or disrupted CDS."""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('Ninja', '1.11.1'),
+    ('Meson', '0.64.0'),
+]
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('Biopython', '1.81'),
+    ('hypothesis', '6.68.2'),
+    ('networkx', '3.0'),
+    ('SciPy-bundle', '2023.02'),
+    ('Pysam', '0.21.0'),
+    ('PyYAML', '6.0'),
+    ('snakemake', '7.32.3'),
+    ('pyfaidx', '0.7.2.1'),
+    ('Greenlet', '2.0.2'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+local_old_line = 'from scipy._build_utils import numpy_nodepr_api'
+local_new_line = """numpy_nodepr_api = dict(define_macros=[("NPY_NO_DEPRECATED_API", "NPY_1_9_API_VERSION")])"""
+# hardcoded value, used to be imported from scipy._build_utils (which is no longer available since scipy 1.10)'
+local_code_patch_cmd = f"sed -i 's/{local_old_line}/{local_new_line}/g' setup.py"
```

F-strings only work with Python 3.6, strictly speaking EasyBuild is still compatible with Python 2, so we shouldn't use them yet:

```suggestion
local_code_patch_cmd = "sed -i 's/%s/%s/g' setup.py" % (local_old_line, local_new_line)